app/views/user_mailer/invitation_to_claim.text.erb

```diff
@@ -22,9 +22,7 @@
 
 <%= t(".unwanted.text", support_link: new_feedback_report_url) %>
 
-<%= t(".redirects") %>
-
-<%= t(".update_redirect.text", open_doors_link: "https://opendoors.transformativeworks.org/en/contact-open-doors/") %>
+<%= t(".redirects.text", faq_link: archive_faqs_url) %>
```

Comment:
This should be `faq_url`, since it's just a URL (per the [URLs section of our i18n Guidelines](https://github.com/otwcode/otwarchive/wiki/Internationalization-(i18n)-Standards#urls))